src/catalog.c

```diff
@@ -111,8 +111,8 @@ const static InternalFunctionDef internal_function_definitions[_MAX_INTERNAL_FUN
 		.args = 3
 	},
 	[DDL_DROP_CHUNK] = {
-		.name = "drop_chunk",
-		.args = 3
+		.name = "drop_chunk_metadata",
```

Comment:
Now the semantics of `DDL_DROP_CHUNK` is unclear, including the DDL-upcall function  `process_drop_chunk`.

Reply:
changed names. Is that what you meant?